### PR TITLE
Automatically port System.IO EII, FileSystemAclExtensions, CopyTo

### PR DESCRIPTION
**Pull request by carlossanlop**
New ACL method here: https://github.com/dotnet/runtime/blob/e1ddf9379dfbd9e740b0120d9d93e0bfb9a8332d/src/libraries/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.cs#L236

New CopyTo method: https://github.com/dotnet/runtime/blob/6b5850fa7cf476b66be9d01a3ed05f4484b01d3a/src/libraries/System.Private.CoreLib/src/System/IO/UnmanagedMemoryStream.cs#L215

### CODE REVIEW

File: xml/System.IO/FileSystemAclExtensions.xml [hunk 1]
```diff
@@ -210,11 +210,21 @@ namespace ConsoleApp
...
+        <param name="directorySecurity">An object that determines the access control and audit security for the directory.</param>
+        <param name="path">The path of the directory to create.</param>
+        <summary>Creates a directory and returns it, ensuring it is created with the specified directory security. If the directory already exists, the existing directory is returned.</summary>
+        <returns>A directory information object representing either a created directory with the provided security properties, or the existing directory.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This extension method was added to .NET Core to bring the functionality that was provided by the `System.IO.Directory.CreateDirectory(System.String,System.Security.AccessControl.DirectorySecurity)` .NET Framework method.
```
Comment (gewarren) [inline xml/System.IO/FileSystemAclExtensions.xml:222]:
```suggestion
This extension method was added to .NET Core to provide the functionality of the .NET Framework method `System.IO.Directory.CreateDirectory(System.String,System.Security.AccessControl.DirectorySecurity)`.
```

Maybe this reads better?

File: xml/System.IO/UnmanagedMemoryStream.xml [hunk 2]
```diff
@@ -646,10 +646,10 @@
         <Parameter Name="bufferSize" Type="System.Int32" Index="2" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="callback">To be added.</param>
-        <param name="state">To be added.</param>
-        <param name="bufferSize">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="callback">The callback tho handle the stream contents to be copied.</param>
```
Comment (gewarren) [inline xml/System.IO/UnmanagedMemoryStream.xml:649]:
```suggestion
        <param name="callback">The callback handle to which the stream contents are sent.</param>
```

File: xml/System.IO/FileSystemAclExtensions.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.IO/FileSystemAclExtensions.xml:222]: Yep, thanks :)

**Comment by carlossanlop** [comment]
rebased to fix merge conflict

**Pull request merged**